package.json

```diff
@@ -48,9 +47,13 @@
         "pretty-quick": "^3.1.3",
         "rimraf": "^3.0.2",
         "run-script-os": "^1.1.6",
+        "turbo": "1.7.4",
         "typescript": "^4.8.4"
     },
     "engines": {
         "node": ">=18.15.0"
+    },
+    "dependencies": {
+        "mysql2": "^3.5.1"
```

Comment:
should be in `packages/components/package.json`

Reply:
Moved it!